stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -1,5 +1,9 @@
 package com.stripe.android;
 
+import android.util.Log;
+
+import java.util.concurrent.Executor;
```

Comment:
@kyleconroy As far as the build dependencies go, I added the revision numbers to remove import warnings Android Studio was giving.  But I don't think anything was broken, so I can bump back down - what do you think?